Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp

```diff
@@ -368,7 +368,7 @@ int CLuaFunctionDefs::DownloadFile(lua_State* luaVM)
                         }
                     }
                 }
-                m_pScriptDebugging->LogCustom(luaVM, 255, 255, 255, "%s: File doesn't exist", lua_tostring(luaVM, lua_upvalueindex(1)));
+                m_pScriptDebugging->LogCustom(luaVM, SString("%s: File doesn't exist", lua_tostring(luaVM, lua_upvalueindex(1))));
```

Comment:
I've noticed this message is unique across the whole codebase in that it used non-prefixed white-colored LogCustom instead of "WARNING: " prepended orange message. I've changed that to be consistent with the rest of the codebase.

Reply:
okay